### PR TITLE
added backupninja support, by default is disable

### PR DESCRIPTION
**Pull request by slimakcz**

### CODE REVIEW

File: metadata/service/support.yml [hunk 1]
```diff
@@ -11,3 +11,5 @@ parameters:
         enabled: true
       sphinx:
         enabled: true
+      backupninja:
```
Comment (fpytloun) [inline metadata/service/support.yml:14]: You should make it as enabled by default.

File: salt/meta/backupninja.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+backup:
+  salt:
+    fs_includes:
+    - /srv/salt/reclass/nodes/_generated
```
Comment (fpytloun) [inline salt/meta/backupninja.yml]: IMO you should backup whole /srv/salt/reclass

Comment (slimakcz) [reply salt/meta/backupninja.yml]: I think this isn't necessary, because usually classes are from git repository.

Comment (fpytloun) [reply salt/meta/backupninja.yml]: I know but it doesn't have to, so I would prefer to backup all.

**Comment by slimakcz** [comment]
@fpytloun ok, by default is enabled: true and change source directory

**Pull request merged**